conslogging/conslogging.go

```diff
@@ -467,11 +468,29 @@ func (cl ConsoleLogger) color(c *color.Color) *color.Color {
 	return noColor
 }
 
+var (
+	// githubRegExp Matches :2dd88e53f2e59e96ec1f9215f24a3981e5565edf+ in a prefix.
+	// 	Prefix containing hash may resemble: g/e/hello-world:2dd88e53f2e59e96ec1f9215f24a3981e5565edf+base
+	//	Prefix must be exactly 40 characters
+	githubRegExp = regexp.MustCompile(`:[a-z0-9]{40}\+`)
+
+	branchOrSHARegExp = regexp.MustCompile(`:[a-zA-Z0-9/\-_]*\+`)
+)
+
 func prettyPrefix(prefixPadding int, prefix string) string {
-	if prefixPadding == NoPadding {
+	if prefixPadding <= NoPadding {
 		return prefix
 	}
 
+	// tempReplacement used to temporarily remove branch name / sha from string
+	const tempReplacement = "{{}}{{{}{}}}"
+	branchOrSHA := branchOrSHARegExp.FindString(prefix)
+	prefix = strings.Replace(prefix, branchOrSHA, tempReplacement, 1)
+	sha := githubRegExp.FindString(branchOrSHA)
+	if sha != "" {
+		branchOrSHA = sha[:7] + "+"
```

Comment:
can we stick to the standard 8 chars (which is what `git rev-parse --short HEAD` gives us)
```suggestion
		branchOrSHA = sha[:8ball: ] + "+"
```

Reply:
Sure - [Vlad had commented before](https://github.com/earthly/earthly/pull/2639#discussion_r1093845118) to bring it down to 7.

Reply:
lol, too many cooks :)

`EARTHLY_GIT_SHORT_HASH` provides 8, which comes from the default git implementation.

but if 7 is picked specifically to match github, then ignore my original comment.

Reply:
done